code-challenges.js

```diff
@@ -17,26 +17,75 @@ const fruit4 = "kiwi"
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
+//These variables set the temperatures to be used as the function arguments
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+//Beginning of the function
+const boilingPoint = (temp) => {
+//First conditional statement returns the necessary string if the temperature is at the boiling point
+  if(temp == 212){
+    return temp + " is at boiling point"
+//Second conditional statement returns the necessary string if the temperature is below the boiling point
+  }else if(temp < 212){
+    return temp + " is below boiling point"
+//Third conditional statement returns the necessary string if the temperature is anything else, which can only be above the boiling point
+  }else {
+    return temp + " is above boiling point"
+  }
+}
+//Command to return function with passed-in argument in the terminal
+console.log(boilingPoint(temp1))
+console.log(boilingPoint(temp2))
+console.log(boilingPoint(temp3))
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
+//These variables establish the two arrays to be combined and measured
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
+//This variable establishes a new array which is a combination of the first two arrays
+const combineArrays = myNumbers1.concat(myNumbers2)
 
+//This function uses the .length method to calculate the number of elements in the combined array established above
+const arrayLength = (length) => {
+  return combineArrays.length
+}
+//This command passes in the function name in order to see the result of the function
+console.log(arrayLength())
```

Comment:
Awesome!  I really like that you turned this into a function.  This could be refactored so all of the steps take place inside of the function making it more dynamic by setting up two parameters and concating the params together inside the function and running .length on that.  To do that, you would also need to pass the variables in as an argument when invoking the function.